wurst/objects/units/TrollBuildings.wurst

```diff
@@ -63,6 +63,9 @@ function createBuilding(int newId, int oldId) returns BuildingDefinition
         ..setTooltipBasic("You can mix herbs inside this pot.")
         ..setTooltipExtended("This pot is so big it is able to hold large amounts of boiling water. Inside, herbs can be mixed into potions.")
         ..setName("Mixing Pot")
+        ..setHitPointsMaximumBase(260)
+        ..setArmorType(ArmorType.Large)
+        ..setDeathTimeseconds(1)
```

Comment:
What happens without this?

Reply:
Pot death animation plays twice because the model animation last 1 seconds. Have to sync it with the actual unit definition.